src/components/expandable/expandable.tsx

```diff
@@ -47,21 +42,12 @@ const SummaryContainer = styled.summary`
 
 const SummaryText = styled.p`
   ${TextStyles.headlineXS}
-  margin: 0 0 0 8px;
+  margin: 0;
+  line-height: 1;
```

Comment:
This fixes the issues with the vertical alignment on Firefox. It is much better than before but still not perfect. Fabian said that it is related to the different typography (with the placement in the line) in Chrome and Firefox. I tried a few things (e.g. `vertical-align`), however it did not get better than now.